src/main/java/nl/knaw/dans/dvcli/command/CollectionCmd.java

```diff
@@ -29,12 +29,13 @@
          mixinStandardHelpOptions = true,
          description = "Manage Dataverse collections (i.e. 'dataverses')")
 public class CollectionCmd extends AbstractSubcommandContainer<DataverseApi> {
+    public CollectionCmd(@NonNull DataverseClient dataverseClient) {
```

Comment:
Correct, constructors should be at the top, not sure if we have some coding style that we should check.

Reply:
Idd, it makes more readable